spec/tictactoe_clojure/game_spec.clj

```diff
@@ -0,0 +1,46 @@
+(ns tictactoe-clojure.game-spec
+  (:require [speclj.core :refer :all]
+            [tictactoe-clojure.board :as board]
+            [tictactoe-clojure.human-player :as human-player]
+            [tictactoe-clojure.computer-player :as computer-player]
+            [tictactoe-clojure.rules :as rules]
+            [tictactoe-clojure.console :as console]
+            [tictactoe-clojure.game :refer :all]))
+
+(describe "tictactoe-clojure.game"
+  (with new-board [1 2 3 4 5 6 7 8 9])
+  (with x-win-1st-row-board ["X" "X" "X" "O" "O" 6 7 8 9])
+  (with board-with-draw ["X" "O" "X" "O" "O" "X" "X" "X" "O"])
+  (with board-displayed "1 | 2 | 3\n---------\n4 | 5 | 6\n---------\n7 | 8 | 9\n")
+
+  (describe "#take-two-moves"
+    (it "places O on the board where specified if it is Os turn"
+      (with-redefs [human-player/human-move (fn [_] 1)
+                    computer-player/optimal-move nil]
+        (should= ["O" 2 3 4 5 6 7 8 9] (take-two-moves @new-board "O")))))
```

Comment:
Same as well, ```take-two-moves``` is only being used by play-game. I can't play a tictactoe with just taking two moves so this should be a private function